ppml/services/kms-utils/docker/entrypoint.sh

```diff
@@ -24,31 +23,31 @@ elif [ "$action" = "generatekeys" ]; then
 	    appkey=$3
 		java -cp $BIGDL_HOME/jars/bigdl-ppml-spark_3.1.2-2.1.0-SNAPSHOT.jar:$SPARK_HOME/jars/*:$SPARK_HOME/examples/jars/*:$BIGDL_HOME/jars/* \
 		com.intel.analytics.bigdl.ppml.examples.GenerateKeys \
-		--primaryKeyPath /home/key/ehsm_encrypted_primary_key \
-		--dataKeyPath /home/key/ehsm_encrypted_data_key \
-		--kmsType EHSMKeyManagementService \
-		--kmsServerIP $EHSM_KMS_IP \
-		--kmsServerPort $EHSM_KMS_PORT \
-		--ehsmAPPID $appid \
-		--ehsmAPPKEY $appkey
+			--primaryKeyPath /home/key/ehsm_encrypted_primary_key \
```

Comment:
Please ensure that Indentation of scrip are the same (2/4 space or tab). Otherwise, it may lead to error during running.

https://google.github.io/styleguide/shellguide.html#indentation

Reply:
Thanks for reminding. I will withdraw this part of the amendment. The changes without verification will not be submitted